test/test_linalg.py

```diff
@@ -44,6 +44,15 @@
 if TEST_SCIPY:
     import scipy
 
+def blaslt_supported_device():
+    if torch.cuda.is_available():
+        if torch.version.hip:
+            for arch in ['gfx90a', 'gfx94']:
```

Comment:
You probably want to use the version itself here, as otherwise we'll need to update this list when we support more GPUs.

Reply:
It's not specific to the HIP version here.  hipblaslt is still a bit early in its development and support is limited to MI200 (gfx90a) and MI300 (gfx940, gfx941, gfx942).  The `torch.version.hip` is a handy way to check if we're a ROCm/HIP PyTorch build or a cuda one.  `torch.version.cuda` is None for ROCm builds, and `torch.version.hip` is None for cuda builds.  Otherwise in test files we adopt the convention of setting the env var PYTORCH_TEST_WITH_ROCM (which sets the var TEST_WITH_ROCM) to use alternative code paths within the test modules, but checking torch.version.hip is often better to use than the env var.